app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    cou = 0
+    for let in phrase:
+        if let.lower() == letter.lower():
+            cou += 1
+    return cou
```

Comment:
Do not use abbreviations or short forms for variable names. Besides that, you could use chaining and built-in string methods to implement this solution in one line.

Reply:
Done